pkg/common/config/config.go

```diff
@@ -0,0 +1,24 @@
+package config
+
+import (
+	"io/ioutil"
+	log "github.com/sirupsen/logrus"
+)
+
+var initContainerTemplate = `
+- name: init-pytorch
+  image: busybox
```

Comment:
Can we add a tag for the image and set IfNotPresent?

Reply:
modified